### PR TITLE
Private registry image support and fixing windows to unix line feed char issue in shell command to retrieve triggers from a function app

### PR DESCRIPTION
**Pull request by SatishRanjan**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by SatishRanjan** [comment]
@ebencarek - can you please take a look at this PR.

**Comment by SatishRanjan** [comment]
@ebencarek - request you to please take a look at it.

File: src/appservice-kube/azext_appservice_kube/custom.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ def update_kube_environment(cmd,
 
 def create_app_service_plan(cmd, resource_group_name, name, is_linux, hyper_v, per_site_scaling=False,
                             app_service_environment=None, kube_environment=None, sku='B1', kube_sku=KUBE_DEFAULT_SKU,
-                            number_of_workers=None, location=None, tags=None, no_wait=False):
+                            number_of_workers=None, location=None, tags=None, no_wait=False):
```
Comment (ebencarek) [inline src/appservice-kube/azext_appservice_kube/custom.py]: Remove trailing whitespace

**Review by ebencarek — CHANGES_REQUESTED** [review]

**Review by ebencarek — APPROVED** [review]

**Comment by SatishRanjan** [comment]
Thanks @ebencarek !

**Pull request merged**